src/main/webapp/app/course/plagiarism-cases/instructor-view/detail-view/plagiarism-case-instructor-detail-view.component.ts

```diff
@@ -137,6 +137,23 @@ export class PlagiarismCaseInstructorDetailViewComponent implements OnInit, OnDe
         this.createdPost.title = `Plagiarism Case ${this.plagiarismCase.exercise!.title}`;
     }
 
+    /**
+     * Get the url segment for different types of exercises.
+     * @param exerciseType type of exercise
+     */
+    getExerciseUrlSegment(exerciseType?: ExerciseType) {
+        switch (exerciseType) {
+            case ExerciseType.TEXT:
+                return 'text-exercises';
+            case ExerciseType.MODELING:
+                return 'modeling-exercises';
+            case ExerciseType.PROGRAMMING:
+                return 'programming-exercises/';
```

Comment:
Unneeded forward slash?
Do we not have a common method for this somewhere?

Reply:
Yes I'll remove the forward slash, thanks 👍 I searched for a common method, there are similar ones but they are also not generic enough.

Reply:
I removed it in https://github.com/ls1intum/Artemis/pull/5544/commits/8352e047a809c951c4daed8fda0f3d3602a403d7